trials/death/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+project(death)
```

Comment:
This is a morbid executable name. I think we should call it "Genocide"

Reply:
:(